tidy3d/plugins/mode/mode_solver.py

```diff
@@ -1472,10 +1474,10 @@ def _center_and_lims(self) -> Tuple[List, List, List, List]:
         _, (h_min_s, v_min_s) = Box.pop_axis(self.simulation.bounds[0], axis=n_axis)
         _, (h_max_s, v_max_s) = Box.pop_axis(self.simulation.bounds[1], axis=n_axis)
 
-        h_min = a_center[n_axis] - self.plane.size[t_axes[0]] / 2
-        h_max = a_center[n_axis] + self.plane.size[t_axes[0]] / 2
-        v_min = a_center[n_axis] - self.plane.size[t_axes[1]] / 2
-        v_max = a_center[n_axis] + self.plane.size[t_axes[1]] / 2
+        h_min = self.plane.center[t_axes[0]] - self.plane.size[t_axes[0]] / 2
+        h_max = self.plane.center[t_axes[0]] + self.plane.size[t_axes[0]] / 2
+        v_min = self.plane.center[t_axes[1]] - self.plane.size[t_axes[1]] / 2
+        v_max = self.plane.center[t_axes[1]] + self.plane.size[t_axes[1]] / 2
 
         h_lim = [
             h_min if abs(h_min) < abs(h_min_s) else h_min_s,
```

Comment:
I noticed you had removed the absolute value here, but now it is restored. Seems like it shouldn't be here?

Reply:
Yes. Initially, I thought the definition of `h_lim` was causing issues in some situations, but I realized that wasn't the case. Since this definition is equivalent to mine without the absolute value, I decided to keep the original. Do you think I should revise it?